### PR TITLE
Eth rollup develop solo acc20

### PR DESCRIPTION
**Pull request by iStrike7**
✄ -----------------------------------------------------------------------------

Thank you for your Pull Request! 🙏 Please make sure it follows the contribution guidelines outlined in
[this document](https://github.com/paritytech/polkadot-sdk/blob/master/docs/CONTRIBUTING.md) and fill
out the sections below. Once you're ready to submit your PR for review, please
delete this section and leave only the text under the "Description" heading.

# Description

*Please include a summary of the changes and the related issue. Please also include relevant motivation and context,
including:*

- What does this PR do?
- Why are these changes needed?
- How were these changes implemented and what do they affect?

*Use [Github semantic
linking](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
to address any open issues this PR relates to or closes.*

Fixes # (issue number, *if applicable*)

Closes # (issue number, *if applicable*)

# Checklist

- [ ] My PR includes a detailed description as outlined in the "Description" section above
- [ ] My PR follows the [labeling requirements](CONTRIBUTING.md#Process) of this project (at minimum one label for `T`
  required)
- [ ] I have made corresponding changes to the documentation (if applicable)
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)

You can remove the "Checklist" section once all have been checked. Thank you for your contribution!

✄ -----------------------------------------------------------------------------

### CODE REVIEW

**Review by goncer — APPROVED** [review]

File: substrate/client/cli/src/commands/run_cmd.rs [hunk 1]
```diff
@@ -149,36 +149,44 @@ pub struct RunCmd {
 	pub keystore_params: KeystoreParams,
 
 	/// Shortcut for `--name Alice --validator` with session keys for `Alice` added to keystore.
-	#[arg(long, conflicts_with_all = &["bob", "charlie", "dave", "eve", "ferdie", "one", "two"])]
+	#[arg(long, conflicts_with_all = &["alice", "bob", "charlie", "dave", "eve", "ferdie", "one", "two"])]
```
Comment (goncer) [inline substrate/client/cli/src/commands/run_cmd.rs:152]: @iStrike7  is there a reason to keep old names? suchh alice, bob,...?

File: substrate/primitives/keyring/src/ethereum.rs [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+//     p: '0x025cdc005b752651cd3f728fb9192182acb3a9c89e19072cbd5b03f3ee1f1b3ffa',
+//     s: '0x7dce9bc8babb68fec1409be38c8e1a52650206a7ed90ff956ae8a6d15eeaaef4', // nosemgrep
+//     type: 'ethereum'
+//   },
+//   {
+//     name: 'Faith',
+//     p: '0x037964b6c9d546da4646ada28a99e34acaa1d14e7aba861a9055f9bd200c8abf74',
+//     s: '0xb9d2ea9a615f3165812e8d44de0d24da9bbd164b65c4f0573e1ce2c8dbd9c8df', // nosemgrep
+//     type: 'ethereum'
+//   }
```
Comment (goncer) [inline substrate/primitives/keyring/src/ethereum.rs:36]: :rocket:

**Review by goncer — APPROVED** [review]
Is sudo address update part of this PR?, am I missing something?

File: substrate/client/cli/src/commands/run_cmd.rs (hunk 1 above)
Comment (iStrike7) [reply substrate/client/cli/src/commands/run_cmd.rs:152]: No reason that I can think of rn, but I'd rather not remove them preemptively... Unless you want me to for some reason

File: substrate/primitives/keyring/src/ethereum.rs (hunk 2 above)
Comment (iStrike7) [reply substrate/primitives/keyring/src/ethereum.rs:36]: Sudo account is now "//Alith". Do you need it to be something else?
https://github.com/mangata-finance/mangata-node/blob/5f7db1920931022da15e7248b0b180603852f8b1/rollup/node/src/chain_spec.rs#L84

Comment (goncer) [reply substrate/primitives/keyring/src/ethereum.rs:36]: all good!

**Pull request merged**